src/Categories/Functor/Power.agda

```diff
@@ -169,7 +169,7 @@ reduce2ʳ : ∀ (G : Bifunctor C C C) {n₁ n₂ n₃} (F₁ : Powerendo n₁) (
 reduce2ʳ G F₁ F₂ F₃ = flattenP-assocʳ $ reduce′ G F₁ $ reduce′ G F₂ F₃
 
 overlaps : (H : Bifunctor D D E) (F G : Powerfunctor′ D I) → Powerfunctor′ E I
-overlaps = overlap-×
+overlaps H = overlap-× H
```

Comment:
Needed because now some implicit parameters appear after H

Reply:
The purpose wasn't just to improve the look of the code, but also to enable usage like 'module H = Bifunctor H' and using all the bifunctor helpers with dotted accessors.  AFAIK there's no way to do that without placing implicits after H.  What I could do is make the 'bare' overlap-× and so on have their own declarations instead of just being exports, that way the implicits could be rearranged all to the front.

Reply:
Great idea - yes please!

Reply:
OK, did that for the two -× operations which seemed to be the ones affected.